appmap/django.py

```diff
@@ -133,25 +133,32 @@ def request_params(request):
     return values_dict(params.lists())
 
 def normalize_path_info(path_info, resolved):
-    # Prior to django 3.2, there doesn't appear to be a good way to
-    # determine whether the pattern that matched a request was a path
-    # or a re_path. To figure it out, just try to use the route as a
-    # regex. If that fails, assume it's a path, and use django's
-    # internal parser to turn it into a regex.
-    regex = resolved.route
-    match = re.match(regex, path_info[1:])
-    if not match:
-        regex = _route_to_regex(resolved.route)[0]
+    if not resolved.kwargs:
+        # No kwargs mean no parameters to normalize
+        return path_info
+
+    # Get the current resolver. Look through its cache of reverse
```

Comment:
Is there a way to use `resolved.tried[-1]` (which should be the pattern that ended up matching) instead of trying to duplicate what `URLResolver.resolve` is doing?

Reply:
No, because `resolved.tried` was introduced in 3.2.

Reply:
Ah, ok, thanks. If there isn't any other way then I suppose this is acceptable, I just hope it isn't too brittle.

Reply:
We could see if `hasattr(resolved, 'tried')` and use it if it's available, falling back to using `reverse_dict` if it's not. The 2.2 implementation seems unlikely to change at this point, and `tried` is part of the 3.2 interface. 

Does the added complexity seem worth it to you?

Reply:
Well, it turns out using `tried` wasn't an option, after all. It's content for included paths isn't helpful. So, I settled on a single implementation.